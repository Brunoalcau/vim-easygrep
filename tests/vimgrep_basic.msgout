
"alphabet.in" 
"alphabet.in" 52L, 104C
  1 %a   "alphabet.in"                  line 1

alphabet.in
(1 of 1): c
  1 %a   "alphabet.in"                  line 3

alphabet.in
(1 of 3): c
[EasyGrep] Result list was saved to 'vimgrep_basic.out' successfully
