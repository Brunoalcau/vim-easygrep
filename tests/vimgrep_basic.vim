
" Load the test data. 
edit alphabet.in

Grep c
let g:EasyGrepIgnoreCase=1
GrepAdd c

ResultListSave vimgrep_basic.out
quit!


