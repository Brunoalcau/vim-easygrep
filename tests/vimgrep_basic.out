alphabet.in|3 col 1| c
alphabet.in|3 col 1| c
alphabet.in|29 col 1| C
